src/routes/organization/[organizationId]/components/OrganizationMetadata.svelte

```diff
@@ -0,0 +1,219 @@
+    confirm('Are you sure you want to remove this member?') &&
+    fetch(`${$page.url.href}/members`, {
+      method: 'DELETE',
+      body: JSON.stringify([uid]),
+    });
+
+  const promoteMember = (uid: string) => {
+    if (!confirm('Are you sure you want to promote this member to an admin?')) return;
+    $organization!.private!.members![uid].role = 'admin';
+  };
+
+  const demoteMember = (uid: string) => {
+    if (!confirm('Are you sure you want to demote this admin to a member?')) return;
+    $organization!.private!.members![uid].role = 'member';
+  };
+
+  const approveInviteRequest = async (uid: string) => {
+    if (Object.keys($organization?.private?.members ?? {}).length >= ($organization?.locked.seats ?? 0)) {
+      alert(
+        '⚠️No Available Seats\n\nYou have reached the seat limit for your organization. Please reach out to support@blendreading.com to increase your seat limit.',
+      );
+    } else {
+      // Don't show the prompt for teams, since they have a high seat limit (256)
+      if ($organization?.locked.isLicensed && !confirm('Are you sure you want to approve this invite request? This will take up an available seat.'))
+        return;
+      inviteRequestsProcessing = [...inviteRequestsProcessing, uid];
+      // We have to do this addition server-side since we need to modify that user's organization list
+      await fetch(`${$page.url.href}/members`, {
+        method: 'POST',
+        body: JSON.stringify({ uids: [uid] }),
+      });
+      inviteRequestsProcessing = inviteRequestsProcessing.filter((id) => id !== uid);
+    }
+  };
+
+  const denyInviteRequest = (uid: string) => {
+    if (!confirm('Are you sure you want to deny this invite request?')) return;
+    $organization!.private!.inviteRequests![uid] = null;
+  };
+</script>
+
+<div class="grid grid-flow-row grid-cols-6 gap-4">
+  {#if $organization?.locked.isLicensed}
+    <div class="card col-span-2">
+      <h2>License End</h2>
+      {$organization.locked.termEnd ? new Date($organization.locked.termEnd).toLocaleDateString() : 'N/A'}
+      <p style="font-size: medium;">
+        To renew or modify your license duration, please email <a href="mailto:support@blendreading.com">support@blendreading.com</a>
+      </p>
+    </div>
+  {/if}
+  <div class={`card ${!$organization?.locked.isLicensed ? 'col-span-3' : 'col-span-2'}`}>
+    <h2>Seats Used</h2>
+    {#if $organization?.locked.isLicensed}
+      {Object.keys($organization.private?.members ?? {}).length}/{$organization.locked.seats}
+      <p style="font-size: medium;">
+        To modify your seat limit, please email <a href="mailto:support@blendreading.com">support@blendreading.com</a>
+      </p>
+    {:else}
+      {Object.keys($organization?.private?.members ?? {}).length}
+      <p style="font-size: medium;">
+        This is a team, which means members do not automatically get Blend Pro licenses. To upgrade to a paid organization, please email <a
```

Comment:
This is a bit wordy I think as well, maybe we say something like 
```suggestion
        Each member of a team is billed individually for Blend Pro. To upgrade to a paid organization, please email <a
```